katana/native/projects/static/doProjectHandling.js

```diff
@@ -0,0 +1,735 @@
+//
+//
```

Comment:
The whole js should be contained in at least one js object
inside the object should have an init function